bob-apps/bin/bobd.rs

```diff
@@ -1,3 +1,20 @@
+#[cfg(all(
```

Comment:
The original idea with `allocator.rs` and `include!` macro was better. I suggest going back to this approach

Reply:
Done.